src/Features/Core/Portable/IncrementalCaches/SymbolTreeInfoIncrementalAnalyzerProvider.cs

```diff
@@ -76,9 +76,25 @@ public MetadataInfo(DateTime timeStamp, SymbolTreeInfo info, HashSet<ProjectId>
 
         public IIncrementalAnalyzer CreateIncrementalAnalyzer(Workspace workspace)
         {
+            var cacheService = workspace.Services.GetService<IWorkspaceCacheService>();
```

Comment:
When the workspace doesn't expose such a service.  An example of this is the workspace we have in EditorTestApp.